osgp/shared/osgp-ws-distributionautomation/src/main/resources/schemas/distributionautomation.xsd

```diff
@@ -233,7 +233,7 @@
                     </xsd:restriction>
                 </xsd:simpleType>
             </xsd:element>
-            <xsd:element name="ModelCode" minOccurs="0" maxOccurs="1">
+            <xsd:element name="ModelCode" minOccurs="1">
```

Comment:
Is model code a required field for the RTU devices? Do we already have a model code available for these kind of devices?

Reply:
There is a check in method `RtuDeviceService.addDeviceModel()` which depends on a manufacturer and model code. Therefore, the field is required.